step.yml

```diff
@@ -0,0 +1,58 @@
+title: |-
+  brew install
+summary: |
+  Install or upgrade formula with brew
+description: |
+  Install or upgrade formula with brew
+website: https://github.com/bitrise-steplib/steps-brew-install
+source_code_url: https://github.com/bitrise-steplib/steps-brew-install
+support_url: https://github.com/bitrise-steplib/steps-brew-install/issues
+host_os_tags:
+  - osx-10.10
+type_tags:
+  - installer
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+
+toolkit:
+  go:
+    package_name: github.com/bitrise-steplib/steps-brew-install
+
+inputs:
+  - packages:
+    opts:
+      title: "Name of the formula"
+      summary: "Name of the formula"
+      description: |
+        Description of this input.
+
+        Can be Markdown formatted text.
+      is_expand: true
+      is_required: true
+      value_options: []
+  - arguments:
+    opts:
+      title: "Arguments for Brew install"
+      summary:  "Arguments for Brew install"
+      description: |
+        Description of this input.
+
+        Can be Markdown formatted text.
+      is_expand: true
+      is_required: false
+      value_options: []
+  - upgrade: yes
+    opts:
+      title: "Upgrade formula if previously installed"
+      summary: "Upgrade formula if previously installed"
+      description: |
+        Description of this input.
```

Comment:
If set the step will upgrade the defined packages by calling `brew reinstall [options] [packages]` command. 
Otherwise the step calls `brew install [options] [packages]`